composer.json

```diff
@@ -4,8 +4,8 @@
 	"license": ["MIT"],
 	"minimum-stability": "dev",
 	"require": {
-		"php": "~7.1",
-		"phpstan/phpstan": "dev-master"
+		"php": "^7.0",
```

Comment:
I would like to use this in a project that uses php7. It doesn't look like this package uses any syntax specific to php 7.1